docs/sources/release-notes/v2-0.md

```diff
@@ -0,0 +1,203 @@
+
+###  TraceQL, the trace query language
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. TraceQL makes it possible to find traces that otherwise would have been difficult or even impossible to identify with Tempo's existing search. It also makes it possible for you to understand the broad context for an event sequence. To understand more about why we built TraceQL and the use cases we think it unlocks, see [our blog post](https://grafana.com/blog/2022/11/30/traceql-a-first-of-its-kind-query-language-to-accelerate-trace-analysis-in-tempo-2.0/).
+
+With this release, we’ve implemented a subset of the full TraceQL language we envision. Users can select traces based on the following:
+
+- Span and resource attributes, timing, and duration
+- Basic aggregates: `count()` and `avg()`
+
+To learn more about the TraceQL syntax, see the TraceQL documentation. For information on how we plan to extend the TraceQL language, see future work.
+
+You can run a TraceQL query either by issuing it to Tempo’s [`search_tag_values_v2` API endpoint](https://grafana.com/docs/tempo/latest/api_docs/#search-tag-values-v2), or, for those using Tempo in conjunction with Grafana, by using Grafana’s TraceQL query editor, described below.
+
+TraceQL requires Tempo’s Parquet columnar format, which is the default block format in Tempo 2.0. For more information, refer to the Tempo [Apache Parquet block format](https://grafana.com/docs/tempo/latest/configuration/parquet/) documentation.
+
+### TraceQL query editor in Grafana
+
+To make it easier to run and visualize the results of TraceQL queries, we’ve added a TraceQL query editor to Grafana’s Tempo data source. The editor is available in Grafana’s Explore interface.
+
+<p align="center"><img src="../traceql/assets/query-editor-results-span.png" alt="Query editor showing span results" /></p>
+
+
+The query editor helps you learn TraceQL by offering autocomplete.
+
+<p align="center"><img src="../traceql/assets/query-editor-auto-complete.png" alt="Query editor showing the auto-complete feature" /></p>
+
+For more information, refer to the [TraceQL query editor documentation](https://grafana.com/docs/tempo/latest/traceql/query-editor.md).
+
+### Apache Parquet block format is the new default
+
+<!-- JOE -- NEED TEST RESULTS -->
+
+The experimental Apache Parquet columnar block format introduced in Tempo v1.5 has been promoted to stable and is now the default trace storage for Tempo 2.0. This format enables increased speed and efficiency for Tempo search.
+In our own environments, we’re seeing that it allows us to scan trace data at rates of 1 TB/s (with some bursts as high as 4 TB/s), a massive improvement over the **XXX** rates that were more typical with our previous storage format. Switching to the Parquet block format also allows users to leverage the large ecosystem of existing Parquet tools and libraries to process and transform their trace data.
```

Comment:
@joe-elliott  Did you see what the rates would be in your research? Need to replace the **XXX** with the correct info.

Reply:
I updated the sentence to information that we had in the 1.5 blog post for data for Parquet. we can update if we get more recent info, but at least this is still an impressive number.